src/LearnositySdk/Utils/Json.php

```diff
@@ -104,4 +110,63 @@ public static function isJson($val)
     {
         return !is_null(json_decode($val, true));
     }
+
+    /**
+     * /**
```

Comment:
Double comment mark.

Reply:
@shtrom comments are fixed now, could you have another look?